app/controllers/v0/health_care_applications_controller.rb

```diff
@@ -72,11 +72,25 @@ def healthcheck
     end
 
     def facilities
-      render(json: lighthouse_facilities_service.get_facilities(lighthouse_facilities_params))
+      lighthouse_facilities = lighthouse_facilities_service.get_facilities(lighthouse_facilities_params)
+
+      if params[:ves_active]
+        active_facilities = ves_active_facilities(lighthouse_facilities)
+
+        render(json: active_facilities)
+      else
+        render(json: lighthouse_facilities)
```

Comment:
⚠️ We'd need to modify the request in `vets-website` `src/applications/hca/components/FormFields/VaMedicalCenter.jsx:17` to pass in this parameter.

If we filtered unconditionally, it'd impact all requests to this endpoint - not sure which default is better?

Reply:
Realistically is there ever going to be a reason to send non active facilities? My inclination would be no? If we do remove that param from the FE, I'd recommend renaming this to `active_facilities` or something just to make it clear that's what this endpoint returns.

Reply:
Matt Long raised the same concern on the Frontend side, and it's a good point. I felt like we should leave the ability to output the full list SOMEhow, but maybe even that doesn't make sense? I'll drop the param.